app/controllers/support_interface/change_course_controller.rb

```diff
@@ -12,6 +12,10 @@ def pick_option
       case change_course_form.change_type
       when 'add_course'
         redirect_to support_interface_add_course_to_application_path(application_form)
+      when 'cancel_course'
+        redirect_to support_interface_cancel_course_path(application_form)
+      when 'cancel_application'
+        redirect_to support_interface_cancel_application_path(application_form)
```

Comment:
"Cancel course" and "Cancel application" makes me think the terminology isn't correct. Anyone better ideas?

Reply:
I guess it's `Cancel {Apply 1|Apply 2} application`? `cancel_application_for_current_phase` for the whole thing? `remove_course`?

Reply:
+1 on remove course

Reply:
remove course sounds better to me too. Cancel seems appropriate for the whole application

Reply:
Ok let's go with "remove" in UI. Would the state still "cancelled" or also "removed"?